pkg/artifactory/resource_artifactory_user.go

```diff
@@ -42,13 +54,13 @@ func resourceArtifactoryUser() *schema.Resource {
 			"admin": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Computed:    true,
+				Default:     false,
 				Description: "When enabled, this user is an administrator with all the ensuing privileges.",
 			},
 			"profile_updatable": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Computed: true,
+				Default:  true,
```

Comment:
This is the default from RT?

Reply:
Yes. Checked based on https://www.jfrog.com/confluence/display/JFROG/Security+Configuration+JSON#SecurityConfigurationJSON-application/vnd.org.jfrog.artifactory.security.User+json